src/test/java/integration/CollectionMethodsTest.java

```diff
@@ -464,4 +471,21 @@ void errorWhenSomeMatchedButNoneShould() {
         "%nExpected: none of elements to match [value==cat] predicate"));
   }
 
+  @Test
+  void shouldItemWithText() {
+    $$("#user-table tbody tr  td.firstname")
+      .shouldBe(itemWithText("Bob"));
+  }
+
+  @Test
+  void errorWhenItemWithTextNotMatchedButShouldBe() {
+    String expectedText = "Luis";
+    assertThatThrownBy(()  -> $$("#user-table tbody tr  td.firstname").shouldBe(itemWithText(expectedText)))
+      .isInstanceOf(TextsMismatch.class)
+      .hasMessageContaining(
+        String.format(String.format("Texts mismatch" +
```

Comment:
I think we should have another error message instead of "Texts mismatch". Something like "element with text not found".

Reply:
Fixed, I have added a new exception class with "Element with text not found" as error message.